index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>A-ONE Quizz app</title>
+    <link
```

Comment:
You are calling your Component library here and on the CSS file as well. If you want to use as import, then there is no need to call it here.

Reply:
this was by miss